recipe/meta.yaml

```diff
@@ -26,31 +32,34 @@ build:
 requirements:
   build:
     - python  # [win]
+    - 7za  # [win]
+    - curl  # [win]
+    - msinttypes  # [win and py27]
     - perl 5.22.2.1
-    - freetype 2.7|2.7.*
-    - fontconfig 2.12.*
+    - freetype 2.7|2.7.*  # [unix]
+    - fontconfig 2.12.*  # [linux]
     - openssl 1.0.*
     - jpeg 9*
     - zlib 1.2.*
     - libpng >=1.6.28,<1.7
-    - gst-plugins-base
+    - gst-plugins-base  # [linux]
     - icu 58.*
-    - libxcb
-    - dbus
+    - libxcb  # [linux]
```

Comment:
Please leave things as they are, `libxcb` is only needed on Linux. The list of requirements to build Qt5 on each platform is listed [here](http://doc.qt.io/qt-5/build-sources.html) and `libxcb` is only mentioned on the Linux seciton.

Qt5 is a big and complex enough package to start meddling with its dependencies just to gain a little bit of consistency.